www/__tests__/useImperialConfig.test.ts

```diff
@@ -0,0 +1,49 @@
+import { convertDistance, convertSpeed, formatForDisplay } from '../js/config/useImperialConfig';
+
+
+// This mock is required, or else the test will dive into the import chain of useAppConfig.ts and fail when it gets to the root
+jest.mock('../js/useAppConfig', () => {
+  return jest.fn(() => ({
+    appConfig: {
+      use_imperial: false
+    },
+    loading: false
+  }));
+});
+  
+describe('formatForDisplay', () => {
+  it('should round to the nearest integer when value is >= 100', () => {
+    expect(formatForDisplay(105)).toBe('105');
+    expect(formatForDisplay(119)).toBe('119');
+  });
+
+  it('should round to 3 significant digits when 1 <= value < 100', () => {
+    expect(formatForDisplay(7.02)).toBe('7.02');
+    expect(formatForDisplay(11.3)).toBe('11.3');
+  });
+
+  it('should round to 2 decimal places when value < 1', () => {
+    expect(formatForDisplay(0.07)).toBe('0.07');
+    expect(formatForDisplay(0.75)).toBe('0.75');
```

Comment:
It doesn't seem like we're actually putting the rounding to the test here. Unit tests should explore the boundaries of possible inputs. So don't be 'safe' with it.

I want to see some trickier inputs, like 105.01 or 99.99 or 0.001 or 0.00001 so we can actually see how the rounding is working in action and make sure it's desirable / as expected.

Reply:
(I know you were probably just using the example inputs I put in the comments. Those are to help illustrate how the function works and these are to test and expose potential bugs. Different inputs for different purposes)

Reply:
I created some hardier tests in my most recent commit:

```javascript
describe('formatForDisplay', () => {
  it('should round to the nearest integer when value is >= 100', () => {
    expect(formatForDisplay(105)).toBe('105');
    expect(formatForDisplay(119.01)).toBe('119');
    expect(formatForDisplay(119.91)).toBe('120');
  });

  it('should round to 3 significant digits when 1 <= value < 100', () => {
    expect(formatForDisplay(7.02)).toBe('7.02');
    expect(formatForDisplay(9.6262)).toBe('9.63');
    expect(formatForDisplay(11.333)).toBe('11.3');
    expect(formatForDisplay(99.99)).toBe('100');
  });

  it('should round to 2 decimal places when value < 1', () => {
    expect(formatForDisplay(0.07178)).toBe('0.07');
    expect(formatForDisplay(0.08978)).toBe('0.09');
    expect(formatForDisplay(0.75)).toBe('0.75');
    expect(formatForDisplay(0.001)).toBe('0');
    expect(formatForDisplay(0.006)).toBe('0.01');
    expect(formatForDisplay(0.00001)).toBe('0');
  });
});
```